src/tests/ftest/datamover/dm_large_file.yaml

```diff
@@ -21,8 +21,8 @@ server_config:
 pool:
   mode: 146
   name: daos_server
-  scm_size: 42000000000       # 42G
-  nvme_size: 700000000000     # 700G
+  scm_size: 95%
```

Comment:
i see the same issue in dm_large_dir.yaml

Reply:
@mchaarawi I updated this to just use percentages in the yaml for both.